e2e/cypress/e2e/tests.ts

```diff
@@ -0,0 +1,11 @@
+export const tests: {
+  description: string;
+  url: string;
+  tablePanelChecks?: string[];
+}[] = [
+  {
+    description: "numeric long 0.1 two-items-by-dimension",
+    url: `https://gist.githubusercontent.com/yesoreyeram/1e749c1cf927f353e7e0edd2ff047396/raw/df1088f5f93874571cf08ffc580f02becfcc61d1/numeric-sample.json`,
+    tablePanelChecks: ["avgSlothCount", "city", "4", "LGA", "7.50", "MIA"],
```

Comment:
i think this makes sense for tables, but for additional panel types i can't think of a way to easily handle it other than versioned screenshots.

maybe updating screenshots could be done with a github action to make it easier to manage?

Reply:
> thinking of validating bar chart / pie chart / timeseries chart via the legend value as seen here

yeah, i think is a good idea for certain types of panels.

the hard part might be for something like timeseries where the behavior differences can happen in the rendering of the graph itself. there might be rendering differences for the lines between points in different versions of grafana depending on if a metric value is undefined (multi frame series) vs null (wide series), so checking the label

Reply:
> these test will give some sanity tests across the panels

yep, agreed. i think it's mostly a question of what are the boundaries/goals of these E2E tests vs the panel tests in grafana OSS. maybe the panel unit tests could be eventually responsible for fully checking rendering compliance with the spec?